### ISSUE TITLE
Importing only "new" strings while importing

### ISSUE THREAD

**Issue by NekoJonez** [GlotPress/GlotPress#1250]
Currently, I'm trying to make the translation gap between nl_BE and nl_NL smaller. So, I download the nl_NL 100% translations to import them in nl_BE. 

Now, I would find it extremely handy if it was possible to only import strings that don't have already a suggestion tied to it OR don't have a translation yet. I would save me some time finishing translations. 

Maybe a checkbox at the import page with "only import untranslated strings" or something along that nature would be handy...

**Comment by amieiro** [comment]
@NekoJonez You could use [WP Polyglots facilitator](https://wp-polyglots-facilitator.jesusamieiro.com/), where you can get the translation in one language from another one with only the untranslated strings.

**Comment by NekoJonez** [comment]
> @NekoJonez You could use [WP Polyglots facilitator](https://wp-polyglots-facilitator.jesusamieiro.com/), where you can get the translation in one language from another one with only the untranslated strings. 

True, but thats external. It would be helpful if its also possible internally. ;)

**Comment by psmits1567** [comment]
Fixed added a checkbox, now it does not import existing translations
Created a PR for it
Fix/skip existing translations on import
#2019
@NekoJonez can you change the title to [Fixed] Importing only "new" strings while importing #1250
